support-frontend/test/selenium/subscriptions/pages/DigitalPackGiftCheckout.scala

```diff
@@ -0,0 +1,25 @@
+package selenium.subscriptions.pages
+
+import org.openqa.selenium.WebDriver
+import selenium.util.Config
+
+class DigitalPackGiftCheckout(implicit val webDriver: WebDriver) extends CheckoutPage {
+  val url = s"${Config.supportFrontendUrl}/subscribe/digital/checkout/gift"
+
+  private val giftRecipientFirstName = id("firstNameGiftRecipient")
```

Comment:
not a blocker, just wondering, should we query these by class and put a "qa-something" class on them?

Reply:
I'd generally favour using IDs over class names; we're gradually phasing out named classes in general as we migrate to Emotion. I did think about using data attributes as QA-specific selectors (eg. data-test="firstNameGiftRecipient"), but I couldn't figure out how to use non-ID or -class selectors with ScalaTest.